packages/medusa/src/services/__tests__/claim.js

```diff
@@ -294,7 +316,9 @@ describe("ClaimService", () => {
     })
 
     it("successfully creates fulfillment", async () => {
-      await claimService.createFulfillment("claim_id", { meta: "data" })
+      await claimService.createFulfillment("claim_id", {
```

Comment:
Necessary because of the "config" object now containing both metadata and noNotification